aws_google_auth/__init__.py

```diff
@@ -277,32 +283,41 @@ def cli():
     parser.add_argument('-S', '--sp-id', default=SP_ID, help='Google SSO SP identifier ($GOOGLE_SP_ID)')
     parser.add_argument('-R', '--region', default=REGION, help='AWS region endpoint ($AWS_DEFAULT_REGION)')
     parser.add_argument('-d', '--duration', default=DURATION, help='Credential duration ($DURATION)')
+    parser.add_argument('-p', '--profile', default=PROFILE, help='AWS profile ($AWS_PROFILE)')
 
     args = parser.parse_args()
 
-    if args.username is None:
-        args.username = raw_input("Google username: ")
-
-    if args.idp_id is None or args.sp_id is None:
-        print "Must set both GOOGLE_IDP_ID and GOOGLE_SP_ID"
-        parser.print_help()
-        sys.exit(1)
-
-    if args.duration is None:
-        print "Setting duration to 3600 seconds"
-        args.duration = 3600
-
     if args.duration > 3600:
         print "Duration must be less than or equal to 3600"
         duration = 3600
 
+    config = prepare.get_prepared_config(
```

Comment:
This looks up the config parameters from the `~/.aws/config` file, overwridden from cli parameters or ENV variables.